include/opendht/dht_proxy_server.h

```diff
@@ -98,6 +98,32 @@ class OPENDHT_PUBLIC DhtProxyServer
 
     asio::io_context& io_context() const;
 
+    struct PushStats {
+        uint64_t highPriorityCount {0};
```

Comment:
It was simpler to add a mutex since this struct gets copied in `updateStats` and that doesn't work with atomic types, hope that's ok but let me know if you'd prefer a different solution.